frontend/packages/topology/src/layouts/BaseLayout.ts

```diff
@@ -115,10 +108,34 @@ class LayoutNode {
     this.yy = undefined;
   }
 
+  @computed
```

Comment:
I don't think this needs to be a `@computed` value because bounds is constantly changing during layout.
Did you do this for performance and find that a memoized value was beneficial?

Reply:
No,  I just assumed that with cloning the performance would be better if we set `@computed`.